### ISSUE TITLE
Migration failing

### ISSUE THREAD

**Issue by renatoalmeidaoliveira** [netbox-community/netbox-napalm#21]
**NetBox version**
Upgrade directly from 3.4.4 to 3.6.1

**Describe the bug**
Seccond migration fails with the following stack trace
https://pastebin.com/AnLh1Yj1

**To Reproduce**
1. Start on 3. 4.4
2. Upgrade directly to 3.6.1
3. Pip install the plugin and add to configuration.py
4. Run migrations for the plugin 
5. See the error

**Expected behavior**
All migration happening with success

**Additional context**
The plugin works even without the seccond migration, but it can be a problem in future relases since migrations are dependent on each other

**Comment by jhofmueller** [comment]
I experienced the same problem today. Here is the traceback of the error when taking the netbox container up:

```
netbox-docker-netbox-1               | ⚙️ Applying database migrations
netbox-docker-netbox-1               | 🧬 loaded config '/etc/netbox/config/configuration.py'
netbox-docker-netbox-1               | 🧬 loaded config '/etc/netbox/config/extra.py'
netbox-docker-netbox-1               | 🧬 loaded config '/etc/netbox/config/local_config.py'
netbox-docker-netbox-1               | 🧬 loaded config '/etc/netbox/config/logging.py'
netbox-docker-netbox-1               | 🧬 loaded config '/etc/netbox/config/plugins.py'
netbox-docker-netbox-1               | Operations to perform:
netbox-docker-netbox-1               |   Apply all migrations: account, admin, auth, circuits, contenttypes, core, dcim, django_rq, extras, ipam, netbox_napalm_plugin, netbox_secrets, sessions, social_django, taggit, tenancy, users, virtualization, wireless
netbox-docker-netbox-1               | Running migrations:
netbox-docker-netbox-1               |   Applying netbox_napalm_plugin.0002_auto_20230215_1752...
netbox-docker-netbox-1               | Traceback (most recent call last):
netbox-docker-netbox-1               |   File "/opt/netbox/netbox/./manage.py", line 10, in <module>
netbox-docker-netbox-1               |     execute_from_command_line(sys.argv)
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
netbox-docker-netbox-1               |     utility.execute()
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/__init__.py", line 436, in execute
netbox-docker-netbox-1               |     self.fetch_command(subcommand).run_from_argv(self.argv)
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/base.py", line 412, in run_from_argv
netbox-docker-netbox-1               |     self.execute(*args, **cmd_options)
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/base.py", line 458, in execute
netbox-docker-netbox-1               |     output = self.handle(*args, **options)
netbox-docker-netbox-1               |              ^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/base.py", line 106, in wrapper
netbox-docker-netbox-1               |     res = handle_func(*args, **kwargs)
netbox-docker-netbox-1               |           ^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/core/management/commands/migrate.py", line 356, in handle
netbox-docker-netbox-1               |     post_migrate_state = executor.migrate(
netbox-docker-netbox-1               |                          ^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/migrations/executor.py", line 135, in migrate
netbox-docker-netbox-1               |     state = self._migrate_all_forwards(
netbox-docker-netbox-1               |
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/migrations/executor.py", line 167, in _migrate_all_forwards
netbox-docker-netbox-1               |     state = self.apply_migration(
netbox-docker-netbox-1               |             ^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/migrations/executor.py", line 252, in apply_migration
netbox-docker-netbox-1               |     state = migration.apply(state, schema_editor)
netbox-docker-netbox-1               |             ^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/migrations/migration.py", line 132, in apply
netbox-docker-netbox-1               |     operation.database_forwards(
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
netbox-docker-netbox-1               |     self.code(from_state.apps, schema_editor)
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/netbox_napalm_plugin/migrations/0002_auto_20230215_1752.py", line 9, in forwards_migrate_napalm
netbox-docker-netbox-1               |     qs = Platform.objects.all().exclude(napalm_driver__exact="")
netbox-docker-netbox-1               |          ^^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1444, in exclude
netbox-docker-netbox-1               |     return self._filter_or_exclude(True, args, kwargs)
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
netbox-docker-netbox-1               |     clone._filter_or_exclude_inplace(negate, args, kwargs)
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/query.py", line 1459, in _filter_or_exclude_inplace
netbox-docker-netbox-1               |     self._query.add_q(~Q(*args, **kwargs))
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1545, in add_q
netbox-docker-netbox-1               |     clause, _ = self._add_q(q_object, self.used_aliases)
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^^
netbox-docker-netbox-1               | ^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
netbox-docker-netbox-1               |     child_clause, needed_inner = self.build_filter(
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |    ^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1426, in build_filter
netbox-docker-netbox-1               |     lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |
netbox-docker-netbox-1               | ^^^^^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1236, in solve_lookup_type
netbox-docker-netbox-1               |     _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
netbox-docker-netbox-1               |
netbox-docker-netbox-1               |                              ^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               | ^^^^^^^^^^^^^^^^^^^^^^^^^
netbox-docker-netbox-1               |   File "/opt/netbox/venv/lib/python3.11/site-packages/django/db/models/sql/query.py", line 1724, in names_to_path
netbox-docker-netbox-1               |     raise FieldError(
netbox-docker-netbox-1               | django.core.exceptions.FieldError: Cannot resolve keyword 'napalm_driver' into field. Choices are: config_template, config_template_id, created, custom_field_data, description, devices, id, last_updated, manufacturer, manufacturer_id, name, napalm, slug, tags, virtual_machines
dependency failed to start: container netbox-docker-netbox-1 is unhealthy
```

**Comment by puhis** [comment]
I'm joining with the same problem. I was trying to add napalm to a freshly installed Netbox version 3.6.1.
Tested on python 3.8 and 3.9. OS - Oracle Linux Server 8.8 and Debian Linux 11.
I think the problem comes from [here ](https://github.com/netbox-community/netbox/issues/13628).
but just in case - here is the traceback of the error:


```
  Applying netbox_napalm_plugin.0002_auto_20230215_1752...Traceback (most recent call last):
  File "/opt/netbox/netbox/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/base.py", line 106, in wrapper
    res = handle_func(*args, **kwargs)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/core/management/commands/migrate.py", line 356, in handle
    post_migrate_state = executor.migrate(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/migrations/executor.py", line 135, in migrate
    state = self._migrate_all_forwards(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/migrations/executor.py", line 167, in _migrate_all_forwards
    state = self.apply_migration(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/migrations/executor.py", line 252, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/migrations/migration.py", line 132, in apply
    operation.database_forwards(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/opt/netbox/venv/lib/python3.9/site-packages/netbox_napalm_plugin/migrations/0002_auto_20230215_1752.py", line 9, in forwards_migrate_napalm
    qs = Platform.objects.all().exclude(napalm_driver__exact="")
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1444, in exclude
    return self._filter_or_exclude(True, args, kwargs)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/query.py", line 1459, in _filter_or_exclude_inplace
    self._query.add_q(~Q(*args, **kwargs))
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1426, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1236, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/opt/netbox/venv/lib/python3.9/site-packages/django/db/models/sql/query.py", line 1724, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'napalm_driver' into field. Choices are: config_template, config_template_id, created, custom_field_data, description, devices, id, last_updated, manufacturer, manufacturer_id, name, napalm, slug, tags, virtual_machines

```

**Comment by arthanson** [comment]
Fixed v0.1.6
